solutions/devsprint-riccardo-1/OnboardingChallenge/Service/Service.swift

```diff
@@ -3,11 +3,14 @@
 import Foundation
 
 protocol ServiceProtocol {
-  func fetchList(_ completion: ([String]) -> Void)
+  func fetchList(_ completion: ([RepositoryCellViewConfiguration]) -> Void)
```

Comment:
This is fine for the moment, but it will change when we will integrate the `ListViewController` with the actual Service.